packages/block-library/src/table/edit.js

```diff
@@ -130,14 +124,8 @@ export default class TableEdit extends Component {
 	 * @param {number} delta Offset for selected row index at which to insert.
 	 */
 	onInsertRow( delta ) {
-		const { selectedCell } = this.state;
-
-		if ( ! selectedCell ) {
-			return;
-		}
-
 		const { attributes, setAttributes } = this.props;
-		const { section, rowIndex } = selectedCell;
+		const { section, rowIndex } = this.state.selectedCell;
```

Comment:
Hm, but `selectedCell` can still be `null`?

Reply:
We check it when providing `isDisabled` for menu items, then they are disabled. In addition, I added return clause in the even callback.